.github/workflows/ci.yml

```diff
@@ -265,6 +265,17 @@ jobs:
           cargo clippy --release --features evm -- -D warnings
           echo "::endgroup::"
 
+      - name: Clean up disk
+        working-directory: ./tee-worker
+        run: |
+          echo "::group::Show disk usage"
+          df -h .
+          echo "::endgroup::"
+          cargo clean --profile release
```

Comment:
why we need `--profile release` here? Can we simply clean all?

Reply:
That's actually the same - we have release profile only anyway